app/javascript/data/minifiedLanguages.json

```diff
@@ -0,0 +1 @@
ya"},{"c":"orm","d":"Oromo"},{"c":"osa","d":"Osage"},{"c":"oss","d":"Ossetian"},{"c":"pag","d":"Pangasinan"},{"c":"pam","d":"Pampanga"},{"c":"pan","d":"Panjabi"},{"c":"pap","d":"Papiamento"},{"c":"pau","d":"Palauan"},{"c":"pol","d":"Polish"},{"c":"pon","d":"Pohnpeian"},{"c":"por","d":"Portuguese"},{"c":"pus","d":"Pushto"},{"c":"que","d":"Quechua"},{"c":"raj","d":"Rajasthani"},{"c":"rap","d":"Rapanui"},{"c":"rar","d":"Rarotongan"},{"c":"roh","d":"Romansh"},{"c":"rom","d":"Romany"},{"c":"ron","d":"Romanian"},{"c":"run","d":"Rundi"},{"c":"rup","d":"Macedo-Romanian"},{"c":"rus","d":"Russian"},{"c":"sad","d":"Sandawe"},{"c":"sag","d":"Sango"},{"c":"sah","d":"Yakut"},{"c":"sas","d":"Sasak"},{"c":"sat","d":"Santali"},{"c":"scn","d":"Sicilian"},{"c":"sco","d":"Scots"},{"c":"sel","d":"Selkup"},{"c":"shn","d":"Shan"},{"c":"sid","d":"Sidamo"},{"c":"sin","d":"Sinhala"},{"c":"slk","d":"Slovak"},{"c":"slv","d":"Slovenian"},{"c":"sma","d":"Southern Sami"},{"c":"sme","d":"Northern Sami"},{"c":"smj","d":"Lule Sami"},{"c":"smn","d":"Inari Sami"},{"c":"smo","d":"Samoan"},{"c":"sms","d":"Skolt Sami"},{"c":"sna","d":"Shona"},{"c":"snd","d":"Sindhi"},{"c":"snk","d":"Soninke"},{"c":"sot","d":"Southern Sotho"},{"c":"sqi","d":"Albanian"},{"c":"srd","d":"Sardinian"},{"c":"srn","d":"Sranan Tongo"},{"c":"srp","d":"Serbian"},{"c":"srr","d":"Serer"},{"c":"ssw","d":"Swati"},{"c":"suk","d":"Sukuma"},{"c":"sun","d":"Sundanese"},{"c":"sus","d":"Susu"},{"c":"swa","d":"Swahili"},{"c":"swe","d":"Swedish"},{"c":"syr","d":"Syriac"},{"c":"tah","d":"Tahitian"},{"c":"tam","d":"Tamil"},{"c":"tat","d":"Tatar"},{"c":"tel","d":"Telugu"},{"c":"tem","d":"Timne"},{"c":"ter","d":"Tereno"},{"c":"tet","d":"Tetum"},{"c":"tgk","d":"Tajik"},{"c":"tgl","d":"Tagalog"},{"c":"tha","d":"Thai"},{"c":"tig","d":"Tigre"},{"c":"tir","d":"Tigrinya"},{"c":"tiv","d":"Tiv"},{"c":"tkl","d":"Tokelau"},{"c":"tli","d":"Tlingit"},{"c":"tmh","d":"Tamashek"},{"c":"tog","d":"Tonga (Nyasa)"},{"c":"ton","d":"Tonga (Tonga Islands)"},{"c":"tpi","d":"Tok Pisin"},{"c":"tsi","d":"Tsimshian"},{"c":"tsn","d":"Tswana"},{"c":"tso","d":"Tsonga"},{"c":"tuk","d":"Turkmen"},{"c":"tum","d":"Tumbuka"},{"c":"tur","d":"Turkish"},{"c":"tvl","d":"Tuvalu"},{"c":"twi","d":"Twi"},{"c":"tyv","d":"Tuvinian"},{"c":"udm","d":"Udmurt"},{"c":"uig","d":"Uighur"},{"c":"ukr","d":"Ukrainian"},{"c":"umb","d":"Umbundu"},{"c":"urd","d":"Urdu"},{"c":"uzb","d":"Uzbek"},{"c":"vai","d":"Vai"},{"c":"ven","d":"Venda"},{"c":"vie","d":"Vietnamese"},{"c":"vot","d":"Votic"},{"c":"wal","d":"Wolaytta"},{"c":"war","d":"Waray (Philippines)"},{"c":"was","d":"Washo"},{"c":"wln","d":"Walloon"},{"c":"wol","d":"Wolof"},{"c":"xal","d":"Kalmyk"},{"c":"xho","d":"Xhosa"},{"c":"yao","d":"Yao"},{"c":"yap","d":"Yapese"},{"c":"yid","d":"Yiddish"},{"c":"yor","d":"Yoruba"},{"c":"zap","d":"Zapotec"},{"c":"zen","d":"Zenaga"},{"c":"zgh","d":"Standard Moroccan Tamazight"},{"c":"zha","d":"Zhuang"},{"c":"zho","d":"Chinese"},{"c":"zul","d":"Zulu"},{"c":"zun","d":"Zuni"},{"c":"zza","d":"Zaza"}]
```

Comment:
Not the biggest fan of this, but if it really is faster I'll deal. How much faster is this than serving from the same list as defined on the backend?

Reply:
Removed and replaced with async functions